solr/core/src/java/org/apache/solr/core/SolrConfig.java

```diff
@@ -810,6 +810,7 @@ public static class UpdateHandlerInfo implements MapSerializable {
     public final boolean openSearcher; // is opening a new searcher part of hard autocommit?
     public final boolean commitWithinSoftCommit;
     public final boolean aggregateNodeLevelMetricsEnabled;
+    public final String commitPollInterval;
```

Comment:
Wondering about string vs. something else here? E.g. something like this maybe:
```suggestion
    public final int commitPollIntervalSeconds;
```
or
```suggestion
    public final Integer commitPollIntervalSeconds;
```

Reply:
Never mind, it is a string where it is e.g. https://github.com/apache/solr/blob/releases/solr/9.7.0/solr/core/src/java/org/apache/solr/handler/ReplicationHandler.java#L251-L253